webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -28,4 +48,146 @@ def calculate_pauschbetrag(has_pflegegrad=False, disability_degree=None, has_mer
     elif disability_degree >= 30:
         return 620
     elif disability_degree >= 20:
-        return 384
\ No newline at end of file
+        return 384
+    
+    return 0
+    
+    
+class PersonAHasDisabilityPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepPersonAHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+
+    @root_validator(skip_on_failure=True)
+    def person_b_has_disability(cls, values):
+        if values.get('person_a_has_disability') != 'yes':
+            raise ValidationError
+        return values
+        
+class PersonBHasDisabilityPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepPersonBHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+
+    @root_validator(skip_on_failure=True)
+    def person_b_has_disability(cls, values):
+        if values.get('person_b_has_disability') != 'yes':
+            raise ValidationError
+        return values
+    
+class StepPauschbetrag(LotseFormSteuerlotseStep):
+    pauschbetrag_prefix = 'person_a'
+        
+    def get_overview_value_representation(self, value):
+        result = ''
+        
+        if value == 'yes':
+            result = self.get_pauschbetrag() + ' ' + _('currency.euro')
+            
+        return result
+
+    def get_pauschbetrag(self):
+        return str(calculate_pauschbetrag(
+            has_pflegegrad=self.stored_data.get(self.pauschbetrag_prefix + '_has_pflegegrad', False),            
+            disability_degree=int(self.stored_data.get(self.pauschbetrag_prefix + '_disability_degree', 0)),   
+            has_merkzeichen_bl=self.stored_data.get(self.pauschbetrag_prefix + '_has_merkzeichen_bl', False),
+            has_merkzeichen_tbl=self.stored_data.get(self.pauschbetrag_prefix + '_has_merkzeichen_tbl', False),
+            has_merkzeichen_h=self.stored_data.get(self.pauschbetrag_prefix + '_has_merkzeichen_h', False)
+        ))
+
+class StepPauschbetragPersonA(StepPauschbetrag):
+    name = 'person_a_requests_pauschbetrag'
+    section_link = SectionLink('mandatory_data', StepFamilienstand.name, _l('form.lotse.mandatory_data.label'))
+
+    preconditions = [PersonAHasDisabilityPrecondition]
+    
+    class InputForm(SteuerlotseBaseForm):
+        person_a_requests_pauschbetrag = SelectField(
+            # TODO: This is empty string mapping of key mapping - the text is within the react component
+            choices=[('yes', 'yes'),('no', '')],
```

Comment:
I don't hink I get the comment.

Reply:
Deleted it :) I just don't like the fact, that we set empty magic strings within the backend and replace them in the frontend...